gslb/cache/controller_obj_cache.go

```diff
@@ -753,13 +753,12 @@ func GetHmDescriptionFromName(hmName string) string {
 }
 
 func GetGSFromHmName(hmName string) (string, error) {
-	// for path based hms
 	hmDesc := GetHmDescriptionFromName(hmName)
-	hmDescriptionSplit := strings.Split(hmDesc, ": ")
-	if len(hmDescriptionSplit) != 5 {
+	hmDescriptionSplit := strings.Split(hmDesc, "gsname: ")
+	if len(hmDescriptionSplit) != 2 {
 		return "", fmt.Errorf("hmName: %s, msg: hm description - \"%s\" is malformed, expected a path based hm", hmName, hmDesc)
```

Comment:
Since this is not meant for path based HMs anymore, can we change this error message?